editor/components/CrashHeader.tsx

```diff
@@ -12,7 +12,9 @@ export default function CrashHeader({ crash }: CrashHeaderProps) {
   return (
     <div className="d-flex justify-content-between mb-3">
       <span className="fs-3 fw-bold text-uppercase">
-        {crash.address_primary}
+        {`${crash.address_primary ? crash.address_primary : ""} ${
+          crash.address_secondary ? "& " + crash.address_secondary : ""
+        }`}
```

Comment:
How likely is it that we wouldnt have an address_primary? I took this code from the old editor, this is how we rendered the title. 

but if we have a secondary address and no primary we'd end up having something like
"& Congress Ave"

Reply:
unlikely, but we have many cases like this

<img width="572" alt="Screenshot 2025-02-06 at 6 30 00 PM" src="https://github.com/user-attachments/assets/3e8868a1-8701-4bb7-bdf0-f4c1a30ec51b" />

which happens because the `rpt_street_name` is `NOT REPORTED`. The missing piece is that we don't include `rpt_hwy_num` in our address concatenation. In the example above it is `290`. 

so long winded way of saying that we need to revisit how we construct the address at the db level. what you have looks good to me 🙌